File: C++/FizzBuzz/FizzBuzz.cpp
#include <cstdio>
#include <cstdlib>
#include <iostream>
using namespace std;
int main(void){
	int max = 2048;
	for (int i=0; i<=max; i++){
		if (i%3 == 0 && i%5 == 0){
				cout << "FizzBuzz";
		}
		else if (i%3 == 0){
				cout << "Fizz";
			}
		else if (i%5 == 0){
				cout << "Buzz";
		}
		else{
				cout << i;
		}
			
		cout << endl;
	}

	return EXIT_SUCCESS;
}

File: Rust/Fizzbuzz/Fizzbuzz.rs
use std::io;

use std::io::prelude::*;
fn main(){
    print!("Input the FizzBuzz Cap: ");
    io::stdout().flush();
    let mut inputText = String::new();
    io::stdin()
        .read_line(&mut inputText)
        .expect("failed to read from stdin");

    let trimmed = inputText.trim();
    match trimmed.parse::<i32>() {
        Ok(i) => fizzbuzz(i),
        Err(..) => println!("Given input: {} was not an integer", trimmed),
}
}
fn fizzbuzz(_max: isize){
    for x in 0.._max {
        if x%5 == 0{
            if x%3 == 0{
                println!("{}", "FizzBuzz");
            }else{
                println!("{}", "Buzz");
		}
        }else if x%3 == 0{
                println!("{}", "Fizz");
        }else{
                println!("{}", x);
        }
    }
    
}
